contrib/OtlpGrpc/Exporter.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+declare(strict_types=1);
+namespace OpenTelemetry\Contrib\OtlpGrpc;
+require __DIR__ . '/../../vendor/autoload.php';
+
+
+use grpc;
+use OpenTelemetry\Sdk\Trace;
+use OpenTelemetry\Trace as API;
+use Opentelemetry\Proto\Collector\Trace\V1;
+
+class Exporter implements Trace\Exporter
+{
+    /**
+     * @var string
+     */
+    private $endpointURL;
+
+    /**
+     * @var string
+     */
+    private $protocol;
+
+    /**
+     * @var string
+     */
+    private $insecure;
+
+    /**
+     * @var string
+     */
+    private $certificateFile;
+
+    /**
+     * @var array
+     */
+    private $headers;
+
+    /**
+     * @var string
+     */
+    private $compression;
+
+    /**
+     * @var int
+     */
+    private $timeout;
+    /**
+     * @var SpanConverter
+     */
+    private $spanConverter;
+    
+    /**
+    * @var bool
+    */
+    private $running = true;
+
+    /**
+     * @var ClientInterface
+     */
+
+    private $client;
+
+    /**
+     * Exporter constructor.
```

Comment:
This comment probably isn't needed, since this is the `__construct` function in the Exporter class.  If you'd like to add a note here for clarity, you'd proabably want it to be `OTLP GRPC Exporter constructor.`.

Reply:
Done